### ISSUE TITLE
Shop list addon

### ISSUE THREAD

**Issue by LankyBox01** [ScratchAddons/ScratchAddons#3415]
### Is your request related to a problem? Please describe.

The normal requests page is being invaded by all of these shops. Proper requests aren't even getting that much attention because of these shops flooding everything.

### Describe the solution you'd like

We should separate every shop from the request and make another page (11/shops) to store all of the shops.
Here's what i did so far (please don't spam my servers with requests):
```
// Copied this function directly from the js docs ;)
async function postData(url = '', data = {}) {
  // Default options are marked with *
  const response = await fetch(url, {
    method: 'POST', // *GET, POST, PUT, DELETE, etc.
    mode: 'cors', // no-cors, *cors, same-origin
    cache: 'no-cache', // *default, no-cache, reload, force-cache, only-if-cached
    credentials: 'same-origin', // include, *same-origin, omit
    headers: {
      'Content-Type': 'application/json'
      // 'Content-Type': 'application/x-www-form-urlencoded',
    },
    redirect: 'follow', // manual, *follow, error
    referrerPolicy: 'no-referrer', // no-referrer, *no-referrer-when-downgrade, origin, origin-when-cross-origin, same-origin, strict-origin, strict-origin-when-cross-origin, unsafe-url
    body: JSON.stringify(data) // body data type must match "Content-Type" header
  });
  return response.text(); // parses JSON response into native JavaScript objects
}

let __SA__ShopUsername = document.getElementsByClassName("user-name")[0].innerText // Get username => https://scratch.mit.edu/discuss/topic/278828/?page=1#post-2849265

// Give manager perms to ELITES :)
if (__SA__ShopUsername == "LankyBox01" || __SA__ShopUsername == "Maximouse" || __SA__ShopUsername == "World_Languages") {
    console.log("(Scratach Addons: Shops add-on): Initiated shop manager mode.") // Just as a log for testing purposes

// A function for adding shops (SHOULD ONLY ADD IF REQUESTED BY THE SHOP OWNER)
function __SA__addShop() {
let __SA__ShopNamePrompt = prompt("Please enter the shop name (Displayed for everyone)..."); // Add tje shop name manually cuz im lazy to make a script to detect it
postData('https://www.sashopdatabase1567842183.tk/createshop', { url: window.location.href, name: __SA__ShopNamePrompt }) // Make a request to my specially-made database (fully secure, don't worry)
  .then(data => {
    console.log(data);
    console.log("Shop added to the list!");
  });
}
}
```

So basically, i made a database full of shops and i wanna make it so that some people can add shops to the database.
And also the shops from the database should be removed from the requests page the user is currently on.

### Describe alternatives you've considered

No alternatives, i guess.

**Comment by RedGuy12** [comment]
...

**Comment by LankyBox01** [comment]
?

**Comment by LankyBox01** [comment]
it's a proper idea, why did you...

**Comment by LankyBox01** [comment]
umm nevermind

**Comment by LankyBox01** [comment]
i'm just gonna go to sleep and wait to realize what i did wrong

**Comment by RedGuy12** [comment]
we're not doing this for reasons that seemed pretty obvious to me

**Comment by wgyt** [comment]
This idea makes no sense for a few reasons:
1. Manual process
2. Why would this be needed? Just ignore the shops
3. This wouldn't add much to the User Experience

**Comment by Joeclinton1** [comment]
What are shops?

**Comment by RedGuy12** [comment]
> What are shops?

https://en.scratch-wiki.info/wiki/Shop_(forums)
